src/modules/Manager.ts

```diff
@@ -121,119 +115,116 @@ export const Manager = (
      * Called on start of user verification. This is when the user ticks the box to claim they are human.
      */
     const start = async () => {
-        try {
-            if (state.loading) {
-                console.log('Procaptcha already loading')
-                return
-            }
-            if (state.isHuman) {
-                console.log('already human')
-                return
-            }
+        // try {
```

Comment:
why has the try catch been commented / removed? It was there to catch any general errors that occur, and without it afaik the errors will be silently displayed on the console.log and procaptcha will become "stuck" in the user's GUI